engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -761,10 +761,17 @@ namespace dmGameSystem
 
         for (uint32_t i = 0; i < textures->m_NumTextures; ++i)
         {
-            uint32_t frame_index = textures->m_Frames[i];
             dmArray<float>& uvs = scratch_uvs[i];
             EnsureSize(uvs, SPRITE_VERTEX_COUNT_SLICE9*2);
 
+            uint32_t frame_index = textures->m_Frames[i];
+            if (frame_index == 0xFFFFFFFF)
+            {
+                // The animation frame wasn't found in the textureset.
```

Comment:
This can happen if for instance the "rename patterns" are wrong, or missing, resulting in mismatching frame names.